native/core/src/events/settings/user.rs

```diff
@@ -39,10 +40,34 @@ impl ActerUserAppSettingsContentBuilder {
         if let Some(chat) = &mut self.chat {
             chat.auto_download = Some(auto_d);
         } else {
+            let typing_notice = self
+                .chat
+                .clone()
+                .expect("No typing notification setting found")
+                .typing_notice;
```

Comment:
the else here means there is no `self.chat` ... this will never work.
```suggestion
```

Reply:
Ohh righttt...